security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -139,6 +154,35 @@ func (s *sdsservice) register(rpcs *grpc.Server) {
 	sds.RegisterSecretDiscoveryServiceServer(rpcs, s)
 }
 
+func (s *sdsservice) DebugInfo() (string, error) {
```

Comment:
Public methods require comment string.

Reply:
Ack. Will also throw some comments over `FetchSecrets`, `StreamSecrets`, and `DeltaSecrets`